post-processing/post-process.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+import os
+import pandas as pd
+from io import StringIO as sio
+import re
+import argparse
+from pathlib import Path
+import glob
+
+""" 
+README
+
+Further documentation is planned for the future. 
+To use this script with the appropriate environment (you'll need Pandas), run this script with:
+
+python post-process.py -path={PATH TO YOUR VERNIER OUTPUTS HERE}
+
+If no path is specified it will look in the current working directory for vernier outputs.
+By default, it will merge the data within the Vernier outputs and write it to the file "vernier-merged-output".
+This can be changed by adding the command-line argument -outputname={YOUR PREFERRED OUTPUTNAME HERE}
+The default vernier files this will attempt to read are of the form 'vernier-output-{rank no}', but this can be changed
+by adding the argument -inputname={YOUR PREFERRED INPUT NAME}
+
+
+README
+""" 
+
+def parse_cli_arguments(input_arguments: list[str] = None,
+                        ) -> argparse.ArgumentParser:
+    """ Parses command line arguments
+
+    Reads in arguments used in the command line and passes them into the parser object.
+
+    Args:
+        input_arguments: A list of command-line arguments to be parsed
+
+    Returns:
+        The 'parser' object. This contains the arguments to be read into variables for later use.
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-path",          type=Path,  default=(os.getcwd()),                help="Path to Vernier output files")
+    parser.add_argument("-outputname",    type=str,   default=str("vernier-merged-output"), help="Name of file to write to")
+    parser.add_argument("-inputname",     type=str,   default=str("vernier-output-"),       help="Vernier files to read from")
+    parser.add_argument("-help",          action="help",                                    help="Show this message and exit")
```

Comment:
As mentioned above we should probably follow a standard for for the optional arguments (e.g. `-p` and `--path` etc.). See [here](https://github.com/MetOffice/lfric_tools/blob/main/bin/cylcReview#L398) for how to do so with argparse.